src/ui-target.c

```diff
@@ -618,9 +617,9 @@ static ui_event target_set_interactive_aux(int y, int x, int mode)
 							/* Save screen */
 							screen_save();
 
-							/* Display */
-							show_floor(floor_list, floor_num,
-									   (OLIST_WEIGHT | OLIST_GOLD), NULL);
+							/* Use OLIST_DEATH to show item labels */
+							show_floor(floor_list, floor_num, (OLIST_DEATH
+									| OLIST_WEIGHT | OLIST_GOLD), NULL);
```

Comment:
I'm using OLIST_DEATH here to force item labels in the object-pile recall. This functionality existed prior, but there were no labels for the items. Looking into it, it seems like OLIST_DEATH is actually just used to force labels, so maybe this isn't a hack so much as OLIST_DEATH could be renamed to OLIST_LABELS?

Reply:
Yes, better.